### PR TITLE
feat: Remove all analytics mentions

### PR DESCRIPTION
**Pull request by paultranvan**
Even though the tracking was already disabled, we experienced validation issues
because the existence of tracking functions calls was confusing.
Therefore, we remove any tracking call to remove this confusion.

### CODE REVIEW

File: src/services/browserPlatformUtils.service.ts [hunk 1]
```diff
@@ -6,16 +6,13 @@ import { DeviceType } from 'jslib/enums/deviceType';
 import { MessagingService } from 'jslib/abstractions/messaging.service';
 import { PlatformUtilsService } from 'jslib/abstractions/platformUtils.service';
 
-// import { AnalyticsIds } from 'jslib/misc/analytics';
-
```
Comment (ptbrowne) [inline src/services/browserPlatformUtils.service.ts:10]: This line can be removed right ?

**Review by ptbrowne — APPROVED** [review]

File: src/services/browserPlatformUtils.service.ts (hunk 1 above)
Comment (paultranvan) [reply src/services/browserPlatformUtils.service.ts:10]: Yes, it can and it is :smile:

Comment (ptbrowne) [reply src/services/browserPlatformUtils.service.ts:10]: Oops, my eyes are not fully opened yet.

**Comment by paultranvan** [comment]
Already merged with in master with #105

**Pull request closed without merging**